custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js

```diff
@@ -1,9 +1,9 @@
 var url = hqImport('hqwebapp/js/initial_page_data').reverse;
 
 function ServiceDeliveryDashboardController($rootScope, $scope, $http, $location, $routeParams, $log, DTOptionsBuilder,
-                                            DTColumnBuilder, $compile, storageService, userLocationId,
-                                            baseControllersService, haveAccessToAllLocations, isAlertActive,
-                                            sddMetadata, dateHelperService, navigationService, isMobile) {
+    DTColumnBuilder, $compile, storageService, userLocationId,
+    baseControllersService, haveAccessToAllLocations, isAlertActive,
+    sddMetadata, dateHelperService, navigationService, isMobile) {
```

Comment:
this is one category of change that i'm never sure how to feel about. just because it puts the function parameters at the same indentation as the function body which i find sometimes trips me up. not sure if that's unique to me or if there's any setting we can change :man_shrugging:

Reply:
I see your point. Looking at [this](https://eslint.org/docs/rules/indent), the `first` option for `parameters` in `FunctionDeclaration` or `FunctionExpression` might do what you want.

I think that would only really affect dashboard devs, since the rest of our js typically uses parameter objects when parameter lists get long. So that sounds like a fine change to make in the global `.eslintrc.js` config file without needing any team discussion about whitespace preferences (since those get contentious...).

Reply:
Thanks, that looks like exactly what I want. https://github.com/dimagi/commcare-hq/pull/26797

(I think I did that right?)

Reply:
looks good!